components/builder-depot/doc/api.raml

```diff
@@ -196,9 +196,34 @@ securitySchemes:
                                         "origin_id": "77731431660388352"
                                     }
 /pkgs:
+    /search:
+        /{query}:
+            get:
+                description: Search for packages with a query string
+                queryParameters:
+                    distinct:
+                        description: Whether to show a distinct list of packages or not
+                        type: boolean
+                        required: false
+                        default: false
+                        example: true
+                responses:
+                    200:
+                        description: Packages were found and fit on one page
+                    204:
+                        description: Packages were found and require pagination
```

Comment:
A 204 means "No content"; does this really return nothing if pagination is required? If pagination _is_ required, what can the user do? There aren't any pagination controls on this endpoint.